src/test/scala/org/apache/mesos/chronos/scheduler/jobs/ScheduleStreamSpec.scala

```diff
@@ -12,17 +12,17 @@ class ScheduleStreamSpec extends SpecificationWithJUnit {
       val orgSchedule = "R3/2012-01-01T00:00:00.000Z/P1D"
       val stream = new ScheduleStream(orgSchedule, null)
       stream.head must_==(orgSchedule, null, "")
-      stream.tail.get.head must_==("R2/2012-01-02T00:00:00.000Z/P1D", null, "")
-      stream.tail.get.tail.get.head must_==("R1/2012-01-03T00:00:00.000Z/P1D", null, "")
-      stream.tail.get.tail.get.tail.get.head must_==("R0/2012-01-04T00:00:00.000Z/P1D", null, "")
-      stream.tail.get.tail.get.tail.get.tail must_== None
+      stream.tail().get.head must_==("R2/2012-01-02T00:00:00.000Z/P1D", null, "")
```

Comment:
I see. It is indeed the appropriate style:

"Scala allows the omission of parentheses on methods of arity-0 (no arguments). However, this syntax should only be used when the method in question has no side-effects (purely-functional). In other words, it would be acceptable to omit parentheses when calling queue.size, but not when calling println(). This convention mirrors the method declaration convention given above."